python/tests/test_almanac.py

```diff
@@ -16,3 +17,32 @@ def test_imports():
 
   """
   assert True
+
+def test_init():
```

Comment:
I figured functional tests of the almanac library would be found in the C tests/ folder whereas these tests would test the python bindings, but now that I look there are no C almanac tests. I'll add a simple test here, but I know @ljbade has #269 which merges the almanac and ephemeris code so that should improve almanac/ephemeris test coverage.